kolibri/core/assets/src/api-resources/task.js

```diff
@@ -1,9 +1,68 @@
 const Resource = require('../api-resource').Resource;
+const logging = require('logging').getLogger(__filename);
 
 class TaskResource extends Resource {
   static resourceName() {
     return 'task';
   }
+
+  localExportContent(driveId) {
+    const promise = new Promise((resolve, reject) => {
+      const clientObj = { path: this.localExportUrl(), entity: { id: driveId },
+        headers: { 'Content-Type': 'application/json' } };
```

Comment:
The mime type interceptor infers what type of data we are sending over? Or is it because we know we are only communicating with json?